src/vulkan-renderer/application.cpp

```diff
@@ -571,17 +531,38 @@ void Application::update_imgui_overlay() {
     m_imgui_overlay->update();
 }
 
-void Application::process_mouse_input() {
-    const auto cursor_pos_delta = m_input_data->calculate_cursor_position_delta();
+void Application::process_input() {
+    const auto cursor_pos_delta = m_input->kbm_data().calculate_cursor_position_delta();
 
-    if (m_camera->type() == CameraType::LOOK_AT && m_input_data->is_mouse_button_pressed(GLFW_MOUSE_BUTTON_LEFT)) {
+    auto deadzone_lambda = [](const float state) {
+        if (state > -0.2f && state < 0.2f) {
```

Comment:
How were these constants chosen? Also it might be easier written as `glm::abs(state) < 0.2f` (at least for me it looks pretty confusing at first)

Reply:
Yeah the constants should be named as `static constexpr WHAT_IS_MY_NAME{0.2f};`